frontend/server/src/Controllers/School.php

```diff
@@ -171,6 +171,25 @@ public static function apiRank(\OmegaUp\Request $r) {
         ];
     }
 
+    /**
+     * Returns rank of best schools in last month
+     *
+     * @param \OmegaUp\Request $r
+     * @return array{time: string, username: string, country_id: string, email: string}[]
+     */
+    public static function apiSchoolCodersOfTheMonth(\OmegaUp\Request $r): array {
+        $r->ensureInt('school_id', null, null, false);
```

Comment:
```suggestion
        $r->ensureInt('school_id');
```

no tiene sentido que se pueda hacer un request sin `school_id`. de cualquier manera arrojaría excepción en L185, y al menos así la excepción arrojada le daría más pistas al que lo llama.